src/components/testimonial/Testimonial.css

```diff
@@ -0,0 +1,121 @@
+.testimonial-card {
+  display: flex;
+  flex-direction: column;
+  border-radius: .3rem;
+  flex-basis: 20rem;

```

Comment:
Ik denk dat de flex-basis wat omhoog mag! Er passen nu 4 items in een rij, terwijl dat er maximaal 3 horen te zijn.